server/api.go

```diff
@@ -556,11 +556,13 @@ func (p *Plugin) getPrsDetails(w http.ResponseWriter, r *http.Request) {
 func fetchPRDetails(ctx context.Context, client *github.Client, prURL string, prNumber int) (*PRDetails, error) {
 	status := ""
 	requestedReviewers := []*string{}
-	var reviewsList []*github.PullRequestReview = nil
+	var reviewsList []*github.PullRequestReview = []*github.PullRequestReview{}
```

Comment:
Would I be correct in assuming you're initializing this so that the resulting JSON serialization returns an empty array `[]` vs the `null` it would otherwise encode when there are no reviews? If that's the case, I'd recommend the more idiomatic:
```
        // Initialize to a non-nil slice to simplify JSON handling semantics.
	reviewsList := []*github.PullRequestReview{}
```

Reply:
It was more a hunch than anything else. I did not like that sometimes reviewsList was a nil, sometimes was a list. I preferred it to be something more stable.